component/portal/src/test/resources/org/exoplatform/portal/config/conf/portal/systemtest/portal.xml

```diff
@@ -20,15 +20,16 @@
 
 <portal-config
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://www.gatein.org/xml/ns/gatein_objects_1_6 http://www.gatein.org/xml/ns/gatein_objects_1_6"
-    xmlns="http://www.gatein.org/xml/ns/gatein_objects_1_6">
+    xsi:schemaLocation="http://www.gatein.org/xml/ns/gatein_objects_1_10 http://www.gatein.org/xml/ns/gatein_objects_1_10"
+    xmlns="http://www.gatein.org/xml/ns/gatein_objects_1_10">
   <portal-name>systemtest</portal-name>
+  <displayed>true</displayed>
```

Comment:
It's value is true, so no need to add it IMO, same for all configurations added even in meeds. BTW, it should have been :
1. either, a field named `hidden` of type `Boolean`: instead of forcing a default value to be `true` (which is not natural in Java and DBMS), we should added a definition of a field which describes the existing data in database (`hidden = false`)
2. or, `visibility` of type Enum Name to be constant with `navigations visibility` field with possible values (`DISPLAYED`, `HIDDEN`, `SYSTEM` or `TEMPORAL`). This will allow more flexibility for future developments. In fact, we have 1 `SYSTEM` site which is `global` and we will add a second one name `public` in a future MIP. 

### **BUT This is just a discussion to let you know about upcoming features in Next MIPs**. No change is request by this comment

Reply:
<displayed> tag removed